sequitur.go

```diff
@@ -259,7 +260,7 @@ func (pr *prettyPrinter) printTerminal(w io.Writer, sym uint64) error {
 	case '\\', '(', ')', '_', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9':
 		out = "\\" + string(sym)
 	default:
-		if _, err := w.Write([]byte{byte(sym)}); err != nil {
+		if _, err := w.Write([]byte(string(rune(sym)))); err != nil {
```

Comment:
or make a bufio.Writer and pass that around? Then you can use w.WriteRune (and probably not a bad idea for performance either, it means callers won't need to remember to do it).

Reply:
I think that's a decision for the repo owner.

Reply:
I like the bufio.Writer idea.

Reply:
I'll leave you to implement it then ;)